SimPEG/EM/Static/Utils/StaticUtils.py

```diff
@@ -943,51 +1350,84 @@ def getSrc_locs(survey):
     return srcMat
 
 
-def gettopoCC(mesh, actind):
+def gettopoCC(mesh, actind, option="top"):
```

Comment:
is there a more descriptive keyword that can be used? `option` is pretty ambiguous

Reply:
topoLoc?

Reply:
topoGridLoc?